src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -337,15 +338,61 @@ public void setFeedbacks(List<Feedback> feedbacks) {
         this.feedbacks = feedbacks;
     }
 
+    /**
+     * Assigns the given feedback list to the result. It first sets the positive flag and the feedback type of every feedback element, clears the existing list of feedback and
+     * assigns the new feedback afterwards.
+     *
+     * @param feedbacks the new feedback list
```

Comment:
I don't really like the method name. It is too close to setFeedbacks and for another developer it is unclear from the method name what is happening. Can we change this, e.g. to updateAllFeedbackItems(...)?

I think we should also make clear that this should not be used for Quizzes and Programming Exercises with completely automatic assessments

Reply:
You are right, this might be misleading. I changed the name and updated the description.